pkg/machines/helpers.es6

```diff
@@ -124,3 +124,17 @@ export function rephraseUI(key, original) {
 
     return transform[key][original];
 }
+
+export function toFixedPrecision(value, precision) {
```

Comment:
This should have a unit test with various formats going on. You can take a look at ```pkg/kdump/test-config-client.js``` for another example of writing basic unit tests for es6 code.